src/configuration/services/oracle-mysql.md

```diff
@@ -0,0 +1,226 @@
+
+## Adjusting MySQL configuration
+
+A select few MySQL configuration properties from the `my.cnf` file are available for adjustment.
+
+At this time, only the `max_allowed_packet` size is available, and defaults to `16` (in MB).  Legal values are from `1` to `100`.
+
+```yaml
+mysqldb:
+    type: oracle-mysql:8.0
+    disk: 2048
+    configuration:
+        properties:
+            max_allowed_packet: 64
+```
+
+The above code will increase the maximum allowed packet size (the size of a query or response) to 64 MB.  However, increasing the size of the maximum packet will also automatically decrease the `max_connections` value.  The number of connections allowed will depend on the packet size and the memory available to the service.  In most cases leaving this value at the default is recommended.
+
+## Access your MySQL service
+
+Assuming your MySQL relationship is named `database`, the host name and port number obtained from `PLATFORM_RELATIONSHIPS` would be `database.internal` and `3306`. Open an [SSH session](/development/ssh.md) and run the MySQL command line client.
+
+```bash
+mysql -h database.internal -P 3306 -u user main
+```
+
+Outside the application container, you can use Platform CLI `platform sql`.
+
+## Exporting data
+
+The easiest way to download all data in a MySQL instance is with the Platform.sh CLI.  If you have a single SQL database, the following command will export all data using the `mysqldump` command to a local file:
+
+```bash
+platform db:dump
+```
+
+If you have multiple SQL databases it will prompt you which one to export. You can also specify one by relationship name explicitly:
+
+```bash
+platform db:dump --relationship database
+```
+
+By default the file will be uncompressed. If you want to compress it, use the `--gzip` (`-z`) option:
+
+```bash
+platform db:dump --gzip
+```
+
+You can use the `--stdout` option to pipe the result to another command. For example, if you want to create a bzip2-compressed file, you can run:
+
+```bash
+platform db:dump --stdout | bzip2 > dump.sql.bz2
+```
+
+## Importing data
+
+The easiest way to load data into a database is to pipe an SQL dump through the `platform sql` command, like so:
+
+```bash
+platform sql < my_database_snapshot.sql
+```
+
+That will run the database snapshot against the SQL database on Platform.sh.  That will work for any SQL file, so the usual caveats about importing an SQL dump apply (e.g., it's best to run against an empty database).  As with exporting, you can also specify a specific environment to use and a specific database relationship to use, if there are multiple.
+
+```bash
+platform sql --relationship database -e master < my_database_snapshot.sql
+```
+
+> **note**
+> Importing a database snapshot is a destructive operation. It will overwrite data already in your database.
+> Taking a snapshot or a database export before doing so is strongly recommended.
+
+## Troubleshooting
```

Comment:
These are all in the troubleshooting page already.  Why duplicate them?  One or the other should just link to the other one.

See also #1104

Reply:
Since this section is identical for both MySQL and MariaDB, I'd say put it in troubleshooting and link both service pages to there.

Reply:
How's this? https://github.com/platformsh/platformsh-docs/pull/1104/commits/66e0b6194c5d6d7f85f40710f24b48d4979f5770

Reply:
Then in oracle and mariadb link to that in a "#Troubleshooting" section at the bottom of the page.